integration/tests/main_test.go

```diff
@@ -0,0 +1,39 @@
+// +build integration
+
+package tests
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	openapi "github.com/asecurityteam/asset-inventory-api/client"
+)
+
+var assetInventoryAPI *openapi.APIClient //this contains pre-configured API client used by all fixtures
+
+func TestMain(m *testing.M) {
+	config := openapi.NewConfiguration()
+	appURL := os.Getenv("AIA_APP_URL")
+	config.BasePath = appURL
+	_, config.Debug = os.LookupEnv("AIA_INTEGRATION_DEBUG")
```

Comment:
This is extremely valuable when debugging integration tests as it sends all interactions with api to stderr verbatim.